vunit/sim_if/ghdl.py

```diff
@@ -364,3 +381,67 @@ def simulate(  # pylint: disable=too-many-locals
             subprocess.call(cmd)
 
         return status
+
+    def _compile_source_file(self, source_file, printer):
+        """
+        Runs parent command for compilation, and moves the .gcno file (if any) to the compilation output folder
+        """
+        compilation_ok = super()._compile_source_file(source_file, printer)
+
+        # GCOV gcno files are output to where the command is run,
+        # move it back to the compilation folder
+        source_path = Path(source_file.name)
+        gcno_file = Path(source_path.stem + ".gcno")
+        if Path(gcno_file).exists():
+            new_path = Path(source_file.library.directory) / gcno_file
+            gcno_file.rename(new_path)
+
+        return compilation_ok
+
+    def merge_coverage(self, file_name, args=None):
+        """
+        Merge coverage from all test cases
+        """
+
+        dir_name = file_name
```

Comment:
Maybe rename to `output_dir` for clarity

Reply:
Done